packages/jaeger-ui/src/components/TracePage/index.test.js

```diff
@@ -80,13 +80,14 @@ describe('makeShortcutCallbacks()', () => {
 });
 
 describe('<TracePage>', () => {
-  TraceTimelineViewer.prototype.shouldComponentUpdate.mockReturnValue(false);
+  TraceTimelineViewer.mockReturnValue('');
```

Comment:
what does this do? I've never mocked the return value of a React component before.